_tests/browser_tests/spec_suites/shared/shared_the-bureau.js

```diff
@@ -16,4 +16,4 @@ describe( 'Beta The Bureau Page', function() {
     expect( page.missions.count() ).toEqual( 3 );
     expect( page.missions.getText() ).toEqual[ 'Educate', 'Enforce', 'Empower' ];
   } );
-} );
+} );
```

Comment:
I just added this too. For @sebworks + @KimberlyMunoz :

- Sublime Text > Preferences > Settings -- User
Should look something like:
```
{
	"font_size": 17,
	"ignored_packages":
	[
		"Vintage"
	],
	"open_files_in_new_window": false,
	"ensure_newline_at_eof_on_save": true
}
```